.gitignore

```diff
@@ -27,12 +27,18 @@
 /tsort
 Makefile
 _FOSSIL_
+cached
 config.status
 config.log
 host-upgrade.sh
+libnv.a
+libucl.a
 poudriere.fossil
 poudriered
 src/bin/poudriere.8.gz
 src/config.h
+src/etc/poudriere.conf
+src/etc/poudriere.d/hooks
```

Comment:
We have valid files checked into this directory.

It's better to just use -e to not use the source's etc dir.